assets/api/api.js

```diff
@@ -77,7 +87,13 @@ function processUpdates(info) {
             }
         }
 
+        if(typeof trackPaused == "function") {
+            let pausedTracks = info.songsOnAir.filter(x => x.isPlaying == false);
+            pausedTracks.forEach(element => trackPaused(element));
+        }
```

Comment:
Hmm, are you getting tick events for paused decks as well? I don't think my setup does this...

Reply:
I guess you could just move it into the code above on the JS side of things. Would it increase the bandwidth of stuff going into the websocket is a whole other question. I'd presume not by a big margin, but don't quote me on that.

Reply:
I meant, move the `is_playing` filter from the Rust side of things into the JS one, which is the only one using it as of now